src/qt/transactionview.cpp

```diff
@@ -396,8 +397,9 @@ void TransactionView::contextualMenu(const QPoint &point)
     abandonAction->setEnabled(model->wallet().transactionCanBeAbandoned(hash));
     bumpFeeAction->setEnabled(model->wallet().transactionCanBeBumped(hash));
 
-    if(index.isValid())
-    {
+    if (index.isValid() && QGuiApplication::platformName() != "minimal") {
+        // The qminimal plugin does not provide window system integration.
+        // Skip this call to suppress warnings during the tests with QT_QPA_PLATFORM=minimal.
         contextMenu->popup(transactionView->viewport()->mapToGlobal(point));
```

Comment:
If you keep this approach maybe refactor this to something like `GUIUtil::popup(contextMenu, ...)` and remove the new include which is already in guiutil.cpp.

Reply:
[Done](https://github.com/bitcoin/bitcoin/pull/17908#issuecomment-599260885).